util/path.go

```diff
@@ -341,15 +347,13 @@ func stripModulePrefixWithCheck(name string) (string, error) {
 // removing foo from "foo:bar". Such qualified paths are used in YANG modules
 // where remote paths are referenced.
 func StripModulePrefix(name string) string {
-	ps := strings.Split(name, ":")
-	switch len(ps) {
-	case 1:
-		return name
-	case 2:
-		return ps[1]
-	default:
-		return name
+	for i, r := range name {
+		if r == ':' && len(name) > i+1 {
```

Comment:
Why is the `len(name) > i+1` necessary? Can this term just be omitted from the if predicate?

Reply:
You are right, this is not needed. I've removed this in the coming commit, thanks.